client/dist/index.html

```diff
@@ -2,6 +2,7 @@
 <html>
   <head>
     <title>BeachRnR</title>
+    <meta name="viewport" content="width=device-width, initial-scale=0.86, maximum-scale=3.0, minimum-scale=0.86">
```

Comment:
Why not just do a simple viewport:

`<meta name="viewport" content="width=device-width, initial-scale=1">`

Reply:
@jbgomez with initial-scale=1 the stars in review section does not render correctly.